std_msgs/Header header

float32 resolution

map_creator/capShape[] capShapes
